analytics_data_api/v0/models.py

```diff
@@ -112,6 +112,25 @@ class Meta(object):
     created = models.DateTimeField(auto_now_add=True)
 
 
+class ProblemResponseAnswerDistribution(BaseProblemResponseAnswerDistribution):
+    """ Original model for the count of a particular answer to a response to a problem in a course. """
+
+    class Meta(BaseProblemResponseAnswerDistribution.Meta):
+        managed = False
+
+    count = models.IntegerField()
+
+
+class ProblemFirstFinalResponseAnswerDistribution(BaseProblemResponseAnswerDistribution):
+    """ Updated model for answer_distribution table with counts of first and final attempts at problems. """
+
+    class Meta(BaseProblemResponseAnswerDistribution.Meta):
+        verbose_name = 'first_last_answer_distribution'
+
+    first_response_count = models.IntegerField()
+    final_response_count = models.IntegerField()
```

Comment:
@dylanrhodes, I chatted with @brianhw and @mulby about this a little.  Would you mind changing the name of this `final_response_count` to `last_response_count` instead?  `ProblemFirstFinalResponseAnswerDistribution` will also need analogous changes.

I realize that I asked for the change to use "final" rather than "last" originally, and I'm very sorry for the back and forth!